codespell_lib/data/dictionary.txt

```diff
@@ -6443,8 +6488,10 @@ instutionalized->institutionalized
 instutions->intuitions
 insufficent->insufficient
 insuffiency->insufficiency
+insure->ensure
```

Comment:
insure is a valid word, see: https://en.oxforddictionaries.com/usage/ensure-or-insure